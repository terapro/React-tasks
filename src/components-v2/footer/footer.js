import React from 'react';
import {Logo} from 'src/components-v2/common/logo/logo';
import {Menu} from 'src/components-v2/menu/menu';    <Menu />

export const Footer = () => (
  <div className='footer'>
    <Logo />
    <Menu />
  </div>
);
