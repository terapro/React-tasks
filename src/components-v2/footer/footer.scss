div.footer {
  display: flex;
  align-items: center;
  flex-direction: column;
}

div.footer ul li {
  margin: $elementPadding auto;
}
