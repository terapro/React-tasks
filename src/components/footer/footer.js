import React from 'react';

import {Logo} from 'src/components/common/logo/logo.js';

const Footer = () => (
        <footer className={'footer'}>
            <Logo />
        </footer>
    );
export {Footer};