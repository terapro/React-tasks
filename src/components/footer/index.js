import React from 'react';

import {Logo} from '../common/logo/logo.js';

const Footer = () => (
        <footer className={'footer'}>
            <Logo />


        </footer>
    );

export {Footer}