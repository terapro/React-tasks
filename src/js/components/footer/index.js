import React from 'react';

import {Logo} from './../common/logo.js';

const Footer = () => (
        <footer className={'footer'}>
            <Logo />


        </footer>
    );

export {Footer}