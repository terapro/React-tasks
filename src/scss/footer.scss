.footer {
  height: 60px;
  width: $appWidth;
  background: #434343;
  flex: 0 0 auto;
  margin: 0 auto;
}

.footer img {
  margin: 20px auto auto 50px;
}