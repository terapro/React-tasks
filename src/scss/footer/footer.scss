.footer {
  height: 60px;
  width: $appWidth;
  background: #434343;
  flex: 0 0 auto;
  margin: 0 auto;
}