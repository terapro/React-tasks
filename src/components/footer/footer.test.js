import React from 'react';
import {Footer} from 'src/components/footer/footer.js';
import {shallow} from 'enzyme';

describe('<Footer />', () => {
  describe('<Footer />', () => {
    it('should render the component', () => {
      const wrapper = shallow(<Footer />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});