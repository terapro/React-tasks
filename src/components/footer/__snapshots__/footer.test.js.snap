// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> <Footer /> should render the component 1`] = `
<footer
  className="footer"
>
  <Logo />
</footer>
`;

exports[`<Footer /> should render the component 1`] = `
<footer
  className="footer"
>
  <Logo />
</footer>
`;
